kmehr/src/main/kotlin/org/taktik/icure/be/ehealth/logic/kmehr/smf/impl/v23g/SoftwareMedicalFileImport.kt

```diff
@@ -416,37 +416,42 @@ class SoftwareMedicalFileImport(
      * @param kmehrIndex the KmehrMessageIndex.
      * @return a List of SubContacts or null.
      */
-    private fun makeSubContact(contactId: String, formId: String?, mfId: String?, service: Service, kmehrIndex: KmehrMessageIndex): List<SubContact>? =
-        kmehrIndex.serviceFor[mfId]
-            ?.mapNotNull { mf -> kmehrIndex.itemIds[mf]?.let { (mf to it) } }
-            ?.map { (heOrHcaMfid, heOrHcaPair) ->
+    private fun makeSubContact(contactId: String, formId: String?, mfId: String?, service: Service, kmehrIndex: KmehrMessageIndex): List<SubContact>? {
```

Comment:
As this function cannot return null anymore, it's better to update the return type
```suggestion
    private fun makeSubContact(contactId: String, formId: String?, mfId: String?, service: Service, kmehrIndex: KmehrMessageIndex): List<SubContact> {
```